app/bundles/EmailBundle/Entity/Email.php

```diff
@@ -42,6 +42,20 @@ class Email extends FormEntity implements VariantEntityInterface, TranslationEnt
     use TranslationEntityTrait;
     use DynamicContentEntityTrait;
 
+    /*
+     * custom variable defined by Eric Jen 6/11/2017.
+     */
```

Comment:
This comment is probably not needed. The authorship and date is logged in the git history.